### ISSUE TITLE
Convert a host to a valid URL automatically

### ISSUE THREAD

**Issue by lirantal** [lirantal/is-website-vulnerable#2]
**Is your feature request related to a problem? Please describe.**
If a host is provided and not a URL with the protocol scheme then lighthouse won't handle it. 
This doesn't work:

```
npx is-website-vulnerable example.com
```

But this does:
```
npx is-website-vulnerable http://example.com
```

**Describe the solution you'd like**
We can convert the website argument provided and turn it into a URL using Node.js' built in API for this, then pass it to lighthouse.

**Comment by ahmedkrmn** [comment]
Can I work on this?

**Comment by SoldierCorp** [comment]
Hi, can I take this one?

**Comment by lirantal** [comment]
Sounds like a plan 👍

**Comment by SoldierCorp** [comment]
I am about to push the changes to my forked repo and I realised you have some guidelines for it so when I run `npm run commit` there isn't any commit script in the package.json, how can I proceed?

**Comment by lirantal** [comment]
@SoldierCorp angular changelog works fine, so things like this should work:
`fix: i fixed something`
`feat(url): auto url coercing`

and so on.
you can also just add a `--no-verify` when you commit or push and I'll just squash the commits from your PR and fix the commit messages before I merge so don't worry too much about it.

**Comment by SoldierCorp** [comment]
Sounds good!

**Comment by lirantal** [comment]
Fixed in #7
